src/main.cpp

```diff
@@ -438,11 +438,13 @@ void setup()
     mqttInit();
 #endif
 
+#ifndef ARCH_PORTDUINO
```

Comment:
It's a thorn that pokes us when we do anything fun

Reply:
plus i don't really get the linker error. network-stubs.cpp should take care of that. That ifnded should not be neccessary. but maybe portduino is 'different' :-)